WooCommerce/Classes/ViewRelated/Hub Menu/HubMenuViewController.swift

```diff
@@ -12,6 +12,12 @@ final class HubMenuViewController: UIHostingController<HubMenu> {
         configureTabBarItem()
     }
 
+    override func viewWillAppear(_ animated: Bool) {
+        // Workaround on iOS 16: hide the navigation bar before showing the hub menu
+        navigationController?.isNavigationBarHidden = true
+        super.viewWillAppear(animated)
```

Comment:
nit: just wondering, any reasons `super` is called the last? I think usually we keep the custom logic after `super` for view will appear/load functions

Reply:
I saw some weird animations on iOS 14 and thought changing this might help the navigation bar to be hidden without the visible lag, but it didn't help 😅.